### DISCUSSION TITLE
address length error

### DISCUSSION THREAD

**Discussion by xiaowh7** [software-mansion/starknet.py#1150] [Q&A] [answered]
when i try to run this code `l0k_pair_contract.functions["balanceOf"].call_sync('0x00fd731dfac1678b0cefb0b50ade21faafd181ecaef1dc193249d41fc5ac91d0', block_number="pending")`
 
get this error `InvalidValueException: Error at path 'account': Shortstring cannot be longer than 31 characters, got: 66.`

but when i use the same address on starkscan, it works.
<img width="728" alt="image" src="https://github.com/software-mansion/starknet.py/assets/5986666/8d389ad3-8766-48ee-80d8-0e277dbc7112">

**Comment by tomek0123456789** [comment, marked as answer]
Hi, you need to pass the argument not as a string, but as an int. You just need to drop quotation marks
